additionalScripts/compileFFmpeg.sh

```diff
@@ -27,13 +27,13 @@ cd ffmpeg-source
 --enable-fontconfig --disable-mips32r2 --disable-mipsdspr2 \
 --disable-htmlpages --disable-podpages --disable-altivec \
 --enable-libass --enable-omx --enable-omx-rpi --enable-libfdk-aac \
---prefix=$(pwd)/ffmpeg-build \
+--prefix="$(pwd)"/ffmpeg-build \
 --pkg-config-flags="--static" --libdir=/usr/local/lib --extra-version=ntd_20150128 --disable-shared --enable-static
 make -j 4
 sudo make install
 
 # Multiline comment, this is how ffmpeg output should look
-: `
+: '
```

Comment:
Learned something new today 😄 

https://stackoverflow.com/questions/43158140/way-to-create-multiline-comments-in-bash